pipeline/matching/evaluation/analytics.py

```diff
@@ -0,0 +1,24 @@
+from typing import Dict, Iterator, List
```

Comment:
How about we rename these files to be metrics instead of analytics?

```
pipeline/matching/evaluation/metrics_test.py
pipeline/matching/evaluation/metrics.py
```